[SOAR-7961] Fix duplicate alerts for devo (#1082)
rapid7/insightconnect-plugins: plugins/devo/help.md

# Version History

> * 3.0.0 - Fix an issue in Get New Alerts trigger which duplicated alerts sending, output changed

Reviewer: ```suggestion
* 3.0.0 - Fix an issue in Get New Alerts trigger to filter duplicate alerts. Output contains list of new alerts
```

# Version History

> * 2.0.0 - Configuration for interval input parameter, changed way of sending alerts by trigger 

Author: Which version should i use? Stay with this or add `3.0.0`?

Reviewer 2: Hey @lukaszlaszuk Because the output schema has changed, it will not be backward-compatible in the workflow builder. A major version bump to 3.0.0 is correct. You'll want to add a new line for 3.0.0 in the version history.

On a separate note, can we improve this error message to be more customer-focused. E.g. It doesn't say anything about what it solves which is fixing an issue in the Get New Alerts trigger that was creating duplicated alerts (..in some config/context) and also call out that the output changed.

Thanks!

Author: Sure, i added version history message for that change
